CHANGELOG.md

```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## Unreleased
+
+### Changed 
+
+- Disabled source with name `references.oats`
```

Comment:
- Field with path `additional_info.catalog.oats.*` not fillable by `references.oats` anymore